roamance/src/api/places-api.ts

```
@@ -38,57 +39,86 @@ export async function getInitialPlaces(): Promise<TouristPlace[]> {
+      const message = errorResponse?.message || 'Failed to fetch popular destinations';
+      throw new ApiError(message, status);
     }
-  });
 
-  console.log(`Successfully fetched ${successCount} out of ${destinations.length} places from Geonames API`);
-  return geonames;
+    throw new ApiError('Failed to fetch popular destinations', 500);
+  }
 }
 
 export async function searchGeonames(query: string, maxRows: number = 10): Promise<Geoname[]> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)